### PR TITLE
Migration of web_tree_dynamic_colored_field to 10.0

### PR DESCRIPTION
**Pull request by jesusVMayor**
Added 9.0 commits from @hbrunn  and @sbidoul , changed version to 10.0 and openerp xml tags to odoo tags.

### CODE REVIEW

File: web_tree_dynamic_colored_field/views/web_tree_dynamic_colored_field.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- vim:fdn=3:
 -->
-<openerp>
+<odoo>
     <data>
```
Comment (MiquelRForgeFlow) [inline web_tree_dynamic_colored_field/views/web_tree_dynamic_colored_field.xml]: remove `data` tags

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

**Comment by jesusVMayor** [comment]
Fixed

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please squash hbrunn's consecutive commits together and I'll merge.

**Comment by jesusVMayor** [comment]
@pedrobaeza done

**Pull request merged**